meta-ov/recipes-apps/ovmenu-ng-skripts/files/transfer-xcsoar.sh

```diff
@@ -20,8 +20,8 @@
 # openvario/backup/<MAC address of eth0>/
 # So you can store backups from more than one OV on the same stick!
 
-echo ' [..........] Starting'
-echo ' [#.........] Wait until "DONE !!" appears before you exit!'
+echo ' [==========] Starting'
+echo ' [#=========] Wait until "DONE !!" appears before you exit!'
```

Comment:
Is this .. an improvement? How? Replace on kind of ASCII art with another? I think the dots are a better "background" graphic, but that's a question of taste.
I think any effort here is useless, because I expect all of this to be replaced with a GUI.

Reply:
Yes, because the menu-x uses proportional font, so we have to change the characters.

Reply:
Wait, this is about displaying a progress bar after the fact from a log file? It doesn't make sense to display a progress bar when the process has already finished, does it?

Reply:
The echo displays line by line in the console during the script's execution. So it really gives an impression of the ongoing progress.